frontend/scripts/react-components/profile/profile-components/table/table.component.jsx

```diff
@@ -198,11 +270,16 @@ class ProfilesTable extends Component {
 
     return (
       <table>
+        {type === 't_head_top_imports' && this.renderTopImportsHeader()}
+        {type === 't_head_top_imports' && this.renderTopImportsTable()}
         {type === 't_head_places' && this.renderPlacesTableHeader()}
```

Comment:
I really don't like this part. I hope we can improve it sometime

Reply:
i agree, i think all this logic should be re-written with re-charts, so we have the same logic as the dashboards